### ISSUE TITLE
How to reduce model training time

### ISSUE THREAD

**Issue by xiunengzy** [SMTorg/smt#361]
...
model = KRG（{}）
model.set_training_values(inputsincolumns,outputsincolumns)
model.train()
...
#============
The amount of data：inputsincolumns（6 rows and 3000 columns），outputsincolumns（7 rows and 3000 columns）
and the trainning takes almost 5mins
Is there some way to increase training speed？

**Comment by relf** [comment]
I suppose you meant 3000 rows with 6 or 7 columns as number of rows should be the same for inputs and outputs. You can try to remove some training samples and use them to validate the surrogate and see if it is good enough.

**Comment by Paul-Saves** [comment]
You could reduce n_start but I do not recommend to do so as it decrease performances.

**Comment by Paul-Saves** [comment]
Ideally, it would be great to fix TNC which is a gradient-based optimizer and train the model much  faster #294 ...
